packages/cookie/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@remix-run/cookie",
+  "version": "0.1.0",
+  "description": "A toolkit for working with cookies in JavaScript",
+  "author": "Michael Jackson <mjijackson@gmail.com>",
```

Comment:
I copied this from `headers` and altered accordingly.

We should probably swap these all over to `hello@remix.run` now that they're migrated into the Remix repo?

Reply:
Yep, that works 👍